cmd/containerd/main.go

```diff
@@ -342,12 +339,13 @@ func loadSnapshotter(store content.Store) (snapshot.Snapshotter, error) {
 		}
 		sn, err := sr.Init(ic)
 		if err != nil {
-			return nil, err
+			log.G(global).WithError(err).Warnf("could not load snapshot plugin %q", name)
```

Comment:
initialization failure is expected, for an example, btrfs fails when /var/lib/containerd/btrfs is not mounted as btrfs.
WDYT?